Cargo.toml

```diff
@@ -39,6 +40,7 @@ travis-ci = { repository = "tokio-rs/tokio" }
 appveyor = { repository = "carllerche/tokio", id = "s83yxhy9qeb58va7" }
 
 [dependencies]
+tokio-codec = { version = "0.1.0", path = "tokio-codec" }
```

Comment:
it should be 0.1.6

Reply:
What’s the thinking here? I noticed that not all of the sub components are on the same version number.

Is it because it’s breaking off of tokio-ui which is 0.1.6?

Reply:
Ah now I think I was wrong